python/docs/source/examples/bluesky.md

```diff
@@ -1,51 +1,74 @@
 # Bluesky Firehose Example
 
-Bluesky is a "distributed social network" that aims to improve on some of the perceived shortcomings of X (nee Twitter). 
-Bluesky uses a distributed protocol name the [AT Protocol](https://atproto.com/) to exchange messages between users, and provides a "firehose" delivering every message sent over the protocol in real-time. 
+Bluesky is a "distributed social network" that aims to improve on some of the perceived shortcomings of X (nee Twitter).
+Bluesky uses a distributed protocol name the [AT Protocol](https://atproto.com/) to exchange messages between users, and provides a "firehose" delivering every message sent over the protocol in real-time.
 
 In this example, we'll show how you can receive and process the firehose using Kaskada.
 
 You can see the full example in the file [bluesky.py](./bluesky.py)
```

Comment:
This version shows a download icon, and downloads it when you click on it.

The second one (referencing github) will link to the file in GitHub. I'd like it to show the GitHub icon, but would need to tweak that if we like that version better.

Pick one after merging?